repose-aggregator/components/services/datastore-service/datastore-service-impl/src/main/java/org/openrepose/nodeservice/distributed/servlet/DistributedDatastoreServlet.java

```diff
@@ -127,7 +139,12 @@ protected void doTrace(HttpServletRequest req, HttpServletResponse resp) throws
         // This servlet is only allowed to communicate with configured Repose instances,
         // so there is no chance of exposing sensitive information.
         // So it is safe to suppress warning squid:S1989
-        resp.sendError(SC_METHOD_NOT_ALLOWED);
+        final Scope scope = startSpan(req, tracer, LOG);
```

Comment:
The `service` method delegates to the individual HTTP verb methods. So my first question is, do we really want to create a new span after we just created a span in `service`? Especially considering that the operation name will already include the method and URI. If we do, still want new spans, then this still doesn't work because:
1. It does not set the span created in `service` as the parent span.
2. It will attempt to create a new span with the same operation name (i.e., method and URI) as the span created in `service`.

Personally, I would be happy to drop the spans created in each individual verb processor method.

Reply:
True. Updated.